src/components/Card/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState }  from 'react';
+import axios from 'axios';
+import './index.css'
+
+
+function Card({user}) {
+    const [repoData, setRepoData] = useState([])
+
+    useEffect(() => {
+        async function fetchRepos() {
+            if(user) {
+                const result = await axios.get(`https://api.github.com/users/${user}/repos`)            
+                console.log(result.data)
+                setRepoData(result.data)
+            } else {
+                return;
+            } 
+        }
+        fetchRepos();
+    },[user])
+
+function display() {
+    let x = document.getElementById('display');
+    if(x.style.display === "block") {
+        x.style.display = "none"
+    } else {
+        x.style.display = "block"
+    }
+}
+
+    return (
+        <>
+        {repoData.map((r) => <div className='cont'>
```

Comment:
When clicking on the element, only the first in the map is hinding the extra info. You could have created a new component f.e. `<DisplayRepo />` and set up a hook inside like this: 
`const [hide, setHide] = useState(false)` or something similar